requirements.txt

```diff
@@ -4,9 +4,9 @@ django-nose==1.3
 logilab-common==0.63.2
 mock==1.0.1
 nose==1.3.4
-pylint==1.4.0
+pylint==1.4.1
 redis==2.10.3
-salt==0.17.1
+salt==2015.8.7
```

Comment:
I disagree, why not match the current version we're using? What if there are major functionality changes between this version and the next?